app/controllers/concerns/client_sortable.rb

```diff
@@ -61,4 +61,17 @@ def limited_user_ids
     val.push(nil) if @filters[:unassigned].present?
     val
   end
-end
\ No newline at end of file
+
+  def normalize_phone_number_if_present(full_query)
+    return if full_query.nil?
+
+    # Regex tested at https://regex101.com/r/4C0dgE/3
```

Comment:
This link gives a detailed breakdown of how the regex is working: https://regex101.com/r/4C0dgE/3/

Reply:
more examples here: https://regex101.com/r/4C0dgE/4